crates/pop-contracts/src/utils/metadata.rs

```diff
@@ -207,7 +354,7 @@ mod tests {
 		assert_eq!(message[2].args[0].label, "new_value".to_string());
 		assert_eq!(message[2].args[0].type_name, "bool".to_string());
 		assert_eq!(message[2].args[1].label, "number".to_string());
-		assert_eq!(message[2].args[1].type_name, "Option".to_string());
+		assert_eq!(message[2].args[1].type_name, "Option<u32>: None, Some(u32)".to_string());
```

Comment:
Whilst this may be unnecessary for Options, showing a list of potential variant values for enums could be very useful imo. This could potentially be refactored into a separate field.